mastodon/src/main/java/org/joinmastodon/android/ui/ImageDescriptionSheet.java

```diff
@@ -0,0 +1,87 @@
+package org.joinmastodon.android.ui;
+
+import android.app.Activity;
+import android.graphics.Typeface;
+import android.graphics.drawable.ColorDrawable;
+import android.os.Build;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.WindowInsets;
+import android.widget.FrameLayout;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.LinearLayoutManager;
+
+import org.joinmastodon.android.R;
+import org.joinmastodon.android.model.Attachment;
+import org.joinmastodon.android.ui.utils.UiUtils;
+
+import me.grishka.appkit.utils.SingleViewRecyclerAdapter;
+import me.grishka.appkit.utils.V;
+import me.grishka.appkit.views.BottomSheet;
+import me.grishka.appkit.views.UsableRecyclerView;
+
+public class ImageDescriptionSheet extends BottomSheet{
+	private UsableRecyclerView list;
+
+	public ImageDescriptionSheet(@NonNull Activity activity, Attachment attachment){
+		super(activity);
+
+		View handleView=new View(activity);
+		handleView.setBackgroundResource(R.drawable.bg_bottom_sheet_handle);
+		ViewGroup handle=new FrameLayout(activity);
+		handle.addView(handleView);
+		handle.setLayoutParams(new FrameLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, V.dp(24)));
```

Comment:
I didn't find where or if the AccountSwitcherSheet sets the height of the handle somewhere - setting it to 24dp manually was the only solution I could find; might be a bit hacky.